run.sh

```diff
@@ -0,0 +1,7 @@
+set -e
+set -x
+
+jekyll build
+aws s3 sync ./_site s3://quay-docs --delete --acl public-read
+aws configure set preview.cloudfront true
+aws cloudfront create-invalidation --distribution-id E26ZSD57Z81FLL --invalidation-batch "{\"Paths\":{\"Quantity\":1,\"Items\":[\"/*\"]},\"CallerReference\":\"jenkins-$(date)\"}"
```

Comment:
We should rotate it anyway if we can (and this discussion is, of course, public)